### PR TITLE
(cloud-merge)[feature] Support to create table with "file_cache_ttl_seconds" property

### PR DESCRIPTION
**Pull request by Lchangliang**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Lchangliang** [comment]
run buildall

**Comment by Lchangliang** [comment]
run buildall

File: be/src/olap/tablet_meta.h [hunk 1]
```diff
@@ -273,6 +273,16 @@ class TabletMeta {
         return _time_series_compaction_level_threshold;
     }
 
+    uint64_t ttl_seconds() const {
+        std::shared_lock rlock(_meta_lock);
+        return _ttl_seconds;
```
Comment (gavinchou) [inline be/src/olap/tablet_meta.h:278]: why the return value is int64? please unify the sign of ttl_seconds to int64

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Comment by Lchangliang** [comment]
run buildall

**Comment by Lchangliang** [comment]
run buildall

**Comment by Lchangliang** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/persist/OperationType.java [hunk 2]
```diff
@@ -225,7 +225,7 @@ public class OperationType {
     // modify table properties: inMemory, StoragePolicy, IsBeingSynced, CompactionPolicy,
     // TimeSeriesCompactionFileCountThreshold, SeriesCompactionTimeThresholdSeconds,
     // SkipWriteIndexOnLoad, EnableSingleReplicaCompaction.
-    public static final short OP_MODIFY_IN_MEMORY = 267;
+    public static final short OP_MODIFY_TABLE_PROPERTIES = 267;
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/persist/OperationType.java:228]: compatibility issue?

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Pull request merged**